scripts/cleanup_old_vms.py

```diff
@@ -160,63 +156,57 @@ def scan_vm(provider_key, vm_name, delta, match_queue, scan_failure_queue):
     # Nested exceptions to try and be safe about the scanned values and to get complete results
     failure = False
     status = NULL
-    logger.info('%r: Scan VM %r...', provider_key, vm_name)
+    vm_creation_time = datetime.datetime(2018, 1, 1, 0, 0).replace(tzinfo=pytz.UTC)
+    logger.info('%r: Scan VM %r...', provider_key, vm.name)
+    # get creation time
     try:
-        vm = get_mgmt(provider_key).get_vm(vm_name)
-        # Localize to UTC
         vm_creation_time = vm.creation_time
     except VMInstanceNotFound:
-        logger.exception('%r: could not locate VM %s', provider_key, vm_name)
+        logger.exception('%r: could not locate VM %s', provider_key, vm.name)
         failure = True
+        pass
     except Exception:  # noqa
+        logger.exception('%r: Exception getting creation time for %r', provider_key, vm.name)
         failure = True
-        logger.exception('%r: Exception getting creation time for %r', provider_key, vm_name)
-        # This VM must have some problem, include in report even though we can't delete
+        # get state of vm that doesn't have creation time
         try:
             status = vm.state
         except Exception:  # noqa
-            failure = True
-            logger.exception('%r: Exception getting status for %r', provider_key, vm_name)
+            logger.exception('%r: Exception getting status for %r', provider_key, vm.name)
             status = NULL
-    finally:
-        if failure:
-            scan_failure_queue.put(VmReport(provider_key, vm_name, FAIL, status, NULL))
-            return
+            pass
```

Comment:
looks like forgotten development bits

Reply:
nah I just always leave `pass` in, but nobody seems to like when I do :)